bandit/formatters/csv.py

```diff
@@ -24,10 +24,10 @@
 .. code-block:: none
 
     filename,test_name,test_id,issue_severity,issue_confidence,issue_text,
-    line_number,line_range
+    more_info,line_number,line_range
     examples/yaml_load.py,blacklist_calls,B301,MEDIUM,HIGH,"Use of unsafe yaml
     load. Allows instantiation of arbitrary objects. Consider yaml.safe_load().
-    ",5,[5]
+    ",https://bandit.readthedocs.io/en/latest/,5,[5]
 
 .. versionadded:: 0.11.0
 
```

Comment:
Can you add 
```
        .. versionchanged:: 1.5.0
              New field `more_info` added to output
```

Since this is a new column and may affect those that are parsing

Reply:
done